docs/features/tuples.work.md

```diff
@@ -0,0 +1,18 @@
+Tuples Work Items
+==================
+
+This is the TODO list for the development of the tuples language feature for C# 7.
+
+# Known issues
+- [ ] Update well-known tuple types to TN naming convention
+- [ ] Generating and loading metadata for user-defined member names
+- [ ] Figure out full behavior for reserved member names
+- [ ] Support tuples 8+
+- [ ] Interop with System.Tuple, KeyValuePair
+- [ ] Publish short-term nuget package for tuples library
+- [ ] Get tuples library into corefx
+- [ ] Debugger bugs
+    - [ ] Tuple debug display is {(1, 2)} because ValueTuple.ToString() returns "(1, 2)"
+    - [ ] Tuple-returning method declaration shows values if names match local variables
+
```

Comment:
semantic info and other IDE stuff
debugger / watch window / expr evaluator./ EnC
various expression contexts 
  expr trees
  dynamic
  unsafe code / pointers
  optional parameters constants 
  xml docs